lizmap/modules/lizmap/classes/qgisForm.class.php

```diff
@@ -381,6 +381,25 @@ public function getForm()
         return $this->form;
     }
 
+    /**
+     * Reset the form controls data to Null.
+     *
+     * @return object the Jelix jForm object
```

Comment:
no, this is not a "jForm" object. `object` is too vague. You should put the real class name, `jFormsBase`

Reply:
Done